sdk/cognitivelanguage/ai-language-text/swagger/README.md

```diff
@@ -132,6 +133,11 @@ directive:
     transform:
       $["x-ms-client-name"] = "LanguageDetectionAction";
       $.description = "Options for a language detection action.";
+  - from: swagger-document
+    where: $.definitions.DynamicClassificationTaskParameters
+    transform:
+      $["x-ms-client-name"] = "DynamicClassificationAction";
+      $.description = "Options for a dynamic classification action.";
```

Comment:
For all these changes, I recommend putting these in an explicit "sharable" section. This is what Anne, Krista, and I have done for CLU and QA. Renames, adding formats, etc., are generally language-agnostic and can be shared if putting them in the swagger itself is not an option. Then we as a v-team can literally copy/paste these definitions and not all have to come to the same eventual conclusion.

/cc @kristapratico @mssfang @joseharriaga

Reply:
I did that before in https://github.com/Azure/azure-sdk-for-js/pull/22298

Reply:
Now that PR is merged, this is naturally done :)